netty/src/test/java/io/grpc/netty/TlsTest.java

```diff
@@ -75,8 +76,8 @@
    */
   @Parameters(name = "{0}")
   public static Iterable<Object[]> data() {
-    return Arrays.asList(new Object[][] {
```

Comment:
It seems you've run an autoformatter on the code. Please revert the changes to existing code that you aren't already touching.

Reply:
Should be fixed now :-)